plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php

```diff
@@ -131,12 +131,12 @@ public function add_settings_page_data( $pages ) {
 
 					// Replace empty string section ids with 'default'.
 					$section_id = '' === $section_id ? 'default' : $section_id;
```

Comment:
Looks like this only needed to be run once for each section. Should we also move this outside of loop?

Reply:
Good call, did that in https://github.com/woocommerce/woocommerce/pull/53016/commits/131b18daef4721ea1b014ad4c18e47a90a3cf499